cluster-service/cspr/orphaned-namespace-cleaner.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: orphaned-namespace-cleaner
+parameters:
+  - name: ORPHANED_NAMESPACE_CLEANER_NAMESPACE
+    description: The namespace to create to have a cronjob which will delete the orphaned namespace which are not deleted due to any issues with the jenkins job.
+    value: orphaned-namespace-cleaner
+  - name: ORPHANED_NAMESPACE_CLEANER_CLUSTERROLE_NAME
+    value: orphaned-namespace-cleaner
+  - name: KUBECTL_IMAGE
+    description: An image which have the `kubectl` binary in it.
+    value: quay.io/rhn_support_ansverma/ubi8-minimal-kubectl:latest
```

Comment:
Hi @geoberle, are we ok with having this image in that registry and repository?

Reply:
I got a reply from @anshulvermapatel that this will be addressed in a separate PR.